### ISSUE TITLE
Switchcraft SSP Wireless modem crash

### ISSUE THREAD

**Issue by kepler155c** [SquidDev-CC/CC-Tweaked#76]
After the last update on the switchcraft server, I am unable to use wireless modems in single player mode.

Adding a wireless modem (ender modem or standard wireless modem) to a neural interface causes Minecraft to crash:

```
net.minecraft.util.ReportedException: Updating screen events
	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1770) ~[bib.class:?]
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1096) ~[bib.class:?]
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:397) [bib.class:?]
	at net.minecraft.client.main.Main.main(SourceFile:123) [Main.class:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_181]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_181]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_181]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_181]
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135) [launchwrapper-1.12.jar:?]
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28) [launchwrapper-1.12.jar:?]
Caused by: java.lang.NoSuchMethodError: dan200.computercraft.shared.peripheral.modem.WirelessModemPeripheral.<init>(Z)V
	at org.squiddev.plethora.integration.computercraft.WirelessModemPeripheralBase.<init>(WirelessModemPeripheralBase.java:31) ~[WirelessModemPeripheralBase.class:?]
	at org.squiddev.plethora.integration.computercraft.WirelessModemPeripheralBase$PeripheralHandler.<init>(WirelessModemPeripheralBase.java:59) ~[WirelessModemPeripheralBase$PeripheralHandler.class:?]
	at org.squiddev.plethora.integration.computercraft.IntegrationComputerCraft$PeripheralCapabilityProvider.getHandler(IntegrationComputerCraft.java:91) ~[IntegrationComputerCraft$PeripheralCapabilityProvider.class:?]
	at org.squiddev.plethora.integration.computercraft.IntegrationComputerCraft$PeripheralCapabilityProvider.hasCapability(IntegrationComputerCraft.java:66) ~[IntegrationComputerCraft$PeripheralCapabilityProvider.class:?]
	at net.minecraftforge.common.capabilities.CapabilityDispatcher.hasCapability(CapabilityDispatcher.java:94) ~[CapabilityDispatcher.class:?]
	at net.minecraft.item.ItemStack.hasCapability(ItemStack.java:1162) ~[aip.class:?]
	at org.squiddev.plethora.gameplay.neural.NeuralItemHandler.isItemValid(NeuralItemHandler.java:42) ~[NeuralItemHandler.class:?]
	at org.squiddev.plethora.gameplay.neural.ContainerNeuralInterface$NeuralSlot.func_75214_a(ContainerNeuralInterface.java:170) ~[ContainerNeuralInterface$NeuralSlot.class:?]
	at net.minecraft.client.gui.inventory.GuiContainer.func_146273_a(GuiContainer.java:469) ~[bmg.class:?]
	at net.minecraft.client.gui.GuiScreen.func_146274_d(GuiScreen.java:548) ~[blk.class:?]
	at org.squiddev.plethora.gameplay.client.gui.GuiNeuralInterface.func_146274_d(GuiNeuralInterface.java:103) ~[GuiNeuralInterface.class:?]
	at net.minecraft.client.gui.GuiScreen.func_146269_k(GuiScreen.java:501) ~[blk.class:?]
	at org.squiddev.plethora.gameplay.client.gui.GuiNeuralInterface.func_146269_k(GuiNeuralInterface.java:58) ~[GuiNeuralInterface.class:?]
	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1757) ~[bib.class:?]
	... 9 more
```

**Comment by SquidDev** [comment]
This should have been fixed by f9c91f288f85da74c8987df51feb0e306cc1a5a5. We haven't updated clients yet, as I want to ensure it's stable first, but you can [download a patched version](https://www.dropbox.com/s/ampikyp544z05zv/cc-tweaked-1.80pr1.10-the-idiot-05.jar?dl=1) for the time being.

**Comment by kepler155c** [comment]
Working as expected in SSP with the patched version. Thanks...
